### PR TITLE
Update node version refs to node 10

### PR DESCRIPTION
**Pull request by Jkovarik**
Update for node10 compat release.      Changes the ecs-task runtime to match the lambda runtime (10.16.3)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,19 +13,12 @@ When included in a Cumulus workflow and deployed to AWS, it will download a spec
 
 ## Compatibility
 
-This only works with Node.js Lambda functions, and requires that the Lambda function it is running has a dependency of at least v1.0.1 of [cumulus-message-adapter-js](https://github.com/cumulus-nasa/cumulus-message-adapter-js).
+This only works with Node.js Lambda functions, and requires that the Lambda function it is running has a dependency of at least v1.1.0 of [cumulus-message-adapter-js](https://github.com/cumulus-nasa/cumulus-message-adapter-js).
```
Comment (markdboyd) [inline README.md]: Out of curiosity, why is `v1.1.0` required?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [v1.4.0]
```
Comment (markdboyd) [inline CHANGELOG.md]: Should we pre-emptively add a new `Unreleased` header above this? I think that's what we do on the main Cumulus repo

File: README.md (hunk 1 above)
Comment (Jkovarik) [reply README.md]: It isn't.  Fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (Jkovarik) [reply CHANGELOG.md]: Possibly.  We tend to immediately release changes on this repo, so I could go either way.

File: README.md [hunk 3]
```diff
@@ -226,21 +86,21 @@ docker run -e AWS_ACCESS_KEY_ID='<aws-access-key>' \
   --lambdaArn <your-lambda-arn>
 ```
 
-Finally, trigger a workflow. You can do this from the Cumulus dashboard, the Cumulus API, or with the AWS Console by supplying a 
+Finally, trigger a workflow. You can do this from the Cumulus dashboard, the Cumulus API, or with the AWS Console by supplying a
```
[the remark this answers is not in the document]
Comment (Jkovarik) [reply README.md]: I'm not sure what Seth intended here.   This section should probably be refactored. 

```suggestion
Finally, trigger a workflow. You can do this from the Cumulus dashboard, the Cumulus API, or with the AWS Console.
```

**Pull request merged**